### PR TITLE
test(journey): add startup settings tests

### PR DESCRIPTION
**Pull request by moriahmaney**

### CODE REVIEW

File: test/journeys/specs/oneOnOne/dataApi/startup-settings.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    before(`create mccoy`, () => testUsers.create({count: 1, config: {displayName: `Bones Mccoy`}})
+    .then((users) => {
+      [mccoy] = users;
+    }));
+
+    before(`pause to let test users establish`, () => browser.pause(5000));
+
+    describe(`initial activity setting`, () => {
+      before(`inject token`, () => {
```
Comment (adamweeks) [inline test/journeys/specs/oneOnOne/dataApi/startup-settings.js]: Maybe create two `describe` blocks for the two different initial activity settings, `meet` or `message`.

File: test/journeys/specs/oneOnOne/dataApi/startup-settings.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        }, spock.token.access_token, mccoy.email);
+        browserLocal.waitForVisible(elements.meetWidget);
+      });
+
+      it(`starts call when set to true`, () => {
+        answer(browserRemote);
+        hangup(browserLocal);
+      });
+
+      after(`refresh browsers to remove widgets`, () => browser.refresh());
```
Comment (adamweeks) [inline test/journeys/specs/oneOnOne/dataApi/startup-settings.js]: Not certain, but you might be able to do
```js
after(`refresh browsers to remove widgets`, browser.refresh);
```

**Comment by bzang** [comment]
What do y'all think about getting this into a hotfix instead of just develop?

**Comment by adamweeks** [comment]
@bzang it could be worthwhile

**Comment by adamweeks** [comment]
Looks good! Can you open another PR of this branch into the `hotfix-20170912` branch as well?

**Comment by moriahmaney** [comment]
@adamweeks yepp, will do now!

**Comment by adamweeks** [comment]
@bzang this PR relies too much on changes in the develop branch, so it would take some major refactoring and cherry picking to get this into a hotfix, I think we'll skip for now

**Pull request merged**